constraint/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Build the manager binary
```

Comment:
Nit: Is this file needed?

Reply:
Probably not. Let me go through and purge things a little more carefully